README.md

```diff
@@ -1,5 +1,5 @@
-# [Welcome to The Odin Project a Open source Web development Curriculum](http://theodinproject.com)
-The Odin project is a open source community and curriculum for learning web development. Odin Project Students complete lessons which include links to the best free resources online and projects which they can use in their portfolio.
+# [Welcome to The Odin Project, an open source web development curriculum](http://theodinproject.com)
+The Odin Project is an open source community and curriculum for learning web development. Odin Project students complete lessons linking to the best free resources online, and projects which they can use in their portfolio.
```

Comment:
How about:

> Students of The Odin Project complete lessons that are constantly curated and updated with links to relevant resources.

I think we can drop the hyperbole (best) and I think the point that TOP is free is covered many times over already haha.